Source/DafnyLanguageServer/Language/Symbols/SymbolTableFactory.cs

```diff
@@ -58,7 +58,25 @@ public class SymbolTableFactory : ISymbolTableFactory {
         .SelectMany(r => r.GetResolvedDeclarations().Select(declaration =>
           ((IDeclarationOrUsage)r, declaration))).ToList();
 
-      return new SymbolTable(usages);
+      var relevantDafnySymbolKinds = new HashSet<DafnySymbolKind>();
+      relevantDafnySymbolKinds.Add(DafnySymbolKind.Function);
+      relevantDafnySymbolKinds.Add(DafnySymbolKind.Class);
+      relevantDafnySymbolKinds.Add(DafnySymbolKind.Enum);
+      relevantDafnySymbolKinds.Add(DafnySymbolKind.Method);
+      relevantDafnySymbolKinds.Add(DafnySymbolKind.EnumMember);
+      relevantDafnySymbolKinds.Add(DafnySymbolKind.Struct);
+      relevantDafnySymbolKinds.Add(DafnySymbolKind.Interface);
+      relevantDafnySymbolKinds.Add(DafnySymbolKind.Namespace);
+      // TODO: Since these definitions are checked for whether they
```

Comment:
Please resolve or remove the TODO. If you want you can leave a comment somewhere to document the inefficiency.